README.md

```diff
@@ -62,3 +62,21 @@ A Repository to collect and showcases developers' favorite resources to learn a
     - The interactivity of the project is what helped it work for me. It is a lot easier to understand something when you're able to actually use it and see the effects.
     - For complete beginners it might be difficult to translate the lessons from this site directly to a real project, especially one where you are collaborating with other people.
 
+  ---
+  
+  
+### [Ming Yong](https://github.com/ming-yong)
+
+> College student learning computer programming in and outside of schools, I am on [Twitter](https://twitter.com/ming__yong).
+
+#### JS/Git/and more
```

Comment:
Hey Ming yong, thanks for adding coding train to the resource list. 

But we are expecting specific blog posts tutorials and videos instead of a whole channel. For eg., Instead of the whole channel add 5 videos you like the most. 

This reduces noise and helps beginners not to get overwhelmed by the load of info

Reply:
Got it! May I change the instruction at the beginning for more clarification as well?

Reply:
Definitely